distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -497,9 +497,6 @@ private static Integer findPortNumber(Integer portToTry) {
                         RMI_SERVER_PORT.getPort()),
                 installStartupFile(getClass().getClassLoader()
                         .getResourceAsStream("hazelcast.xml"), "/etc/hazelcast.xml"),
-                installStartupFile(getClass().getClassLoader()
```

Comment:
Yeah, that's the behavior I've observed. 

If the distribution does **not** have a `ddf.security.sts.client.configuration.config` file by default, then one is created and the `address` line is added inside `test-itests-ddf/target/exam/<GUID>/etc` with a resulting file of
```
address="https://localhost:20002/services/SecurityTokenService?wsdl"
```

If the distribution does have a `ddf.security.sts.client.configuration.config` file of the following form by default (by placing it inside `distribution/ddf-common/src/main/resources` and building)
```
address="https://willbereplaced.url"
useKey=B"false"
```
Then the resulting file inside `test-itests-ddf/target/exam/<GUID>/etc` is
```
address="https://localhost:20002/services/SecurityTokenService?wsdl"
useKey=B"false"
```